test/tokens/XcertMock.test.js

```diff
@@ -26,13 +26,6 @@ contract('XcertMock', (accounts) => {
     await xcert.setPause(true);
     await assertRevert(xcert.transferFrom(accounts[0], accounts[1], id1));
   });
-
-  it('correctly chains additional proof', async () => {
-    await xcert.chain(id1, mockProof2);
-    var proof = await xcert.tokenProof(id1);
-    assert.equal(proof, mockProof2);
-  });
-
```

Comment:
https://github.com/0xcert/ethereum-xcert/pull/6/commits/37c9f2b0c0c03f4c486e43ad50c0a871806493a2, https://github.com/0xcert/ethereum-xcert/pull/6/commits/f5170c7c0c8958c858ba1ce31e6bf1612a89aea7 and this commit should be joined into one. What we want is that every change to the codebase (every commit) passes the tests. So if you selectively remove the code then some repo states become broken.